shared/teams/team/settings-tab/invite-links.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Container from '../../../util/container'
+import * as Types from '../../../constants/types/teams'
+import * as Constants from '../../../constants/teams'
+import * as Styles from '../../../styles'
+import * as dateFns from 'date-fns'
+
+type Props = {
+  teamID: Types.TeamID
+}
+const formatNicely = (etime: number) => (etime === 0 ? 'after 10,000 ys' : dateFns.formatDistanceToNow(etime))
```

Comment:
put in `util/timestamp`. also the designs lack "about". also it says "about 50 years" for me when it actually expires much sooner.

Reply:
Drat, I forgot to multiply by 1000